nixos/doc/manual/man-nixos-build-vms.8

```diff
@@ -0,0 +1,112 @@
+.\" TODO
+.\" .Ql for literals
+.\" .D1 for inline commands
+.Dd January 1, 1980
+.\" nixpkgs groff will use Nixpkgs as the OS in the title by default, taking it from
+.\" doc-default-operating-system. mandoc doesn't have this register set by default,
+.\" so we can use it as a groff/mandoc switch.
+.ie ddoc-default-operating-system .Dt nixos-build-vms \&8 "NixOS System Manager's Manual"
+.el .Dt nixos-build-vms 8
+.Os NixOS
+.Sh NAME
+.Nm nixos-build-vms
+.Nd build a network of virtual machines from a network of NixOS configurations
+.
+.
+.
+.Sh SYNOPSIS
+.Nm nixos-build-vms
+.Op Fl -show-trace
```

Comment:
Every time you have "Fl -" like this, you can instead write "Fl Fl", which maybe looks a bit strange. (But I don't mind if you like it better this way.)